### ISSUE TITLE
light-client-verifier does not build

### ISSUE THREAD

**Issue by S1nus** [celestiaorg/celestia-tendermint-rs#28]
### What went wrong?
Cannot build the light-client-verifier (`v0.32.1` tag) package on macOS with `cargo 1.80.0-nightly (0ca60e940 2024-05-08)`. 

### Steps to reproduce

Attempt to build the light-client-verifier package
```
git clone https://github.com/eigerco/celestia-tendermint-rs.git
cd celestia-tendermint-rs/light-client-verifier
cargo build
```
Observe this error, despite the presence of the `../tendermint` crate with seemingly-valid `Cargo.toml` manifest
```
warning: `/Users/connorohara/Prog/celestia-tendermint-rs/.cargo/config` is deprecated in favor of `config.toml`
note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
    Updating crates.io index
error: no matching package named `tendermint` found
location searched: /Users/connorohara/Prog/celestia-tendermint-rs/tendermint
required by package `tendermint-light-client-verifier v0.32.0 (/Users/connorohara/Prog/celestia-tendermint-rs/light-client-verifier)`
```

### Definition of "done"

Understand why it does not build

**Comment by zvolin** [comment]
I agree this should be mentioned in readme, I thought I documented it somewhere. We only support [celestia-tendermint](https://crates.io/crates/celestia-tendermint) and [celestia-tendermint-proto](https://crates.io/crates/celestia-tendermint-proto). We currently just lack expertise/resources to have a working fork of tendermint on rust side and we are thinking to deprecate even those two.
[light-client-verifier](https://github.com/eigerco/celestia-tendermint-rs/blob/v0.32.x-celestia/light-client-verifier/Cargo.toml#L18) fallls out of our scope

**Comment by S1nus** [comment]
gotcha
